upgrade_task.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE events_to_track (
```

Comment:
There is no benefit to using the varying(36)` to limit the key length to `36` chars - use `varying(255)` so we won't have to touch it future forward.

Reply:
Do we want soft tables?
I do not see any primary key, a search here will kill the DB